help/de/Help_ch_Account-Actions.xml

```diff
@@ -701,90 +803,121 @@
         <orderedlist spacing="compact">
           <listitem>
             <para>&app; bedient sich eines externen Tools zum Abrufen von Online-Kursen. Dieses Tool ist ein
-              Perl-Modul namens <quote>Finance::Quote</quote> und muss unabhängig von &app; auf
-              Ihrem Computer installiert werden. Als erster Schritt zur Benutzung der Online-
+              Perl-Modul namens &app-fq; und muss unabhängig von &app; auf Ihrem Computer
+              installiert werden.
+<!-- Als erster Schritt zur Benutzung der Online-
               Preisaktualisierung sollte daher sein, sicherzustellen, dass
-              <application>Finance::Quote</application> korrekt installiert ist.
+              &app-fq; korrekt installiert ist.  -->
             </para>
 
-            <para>Um festzustellen, ob das Perl-Modul <application>Finance::Quote</application> bereits auf Ihrem
-              System installiert ist, geben Sie <code>perldoc Finance::Quote</code> in ein
+            <tip>
+              <para>Die Installation wird in den
+                <ulink url="&url-docs-build-de;guide/invest-stockprice1.html#invest-stockprice-auto2">&app;
+                Tutorial und Konzepte</ulink> beschrieben.
+              </para>
+            </tip>
+<!-- FIXME source: delete this list. The installation of F::Q is described in the guide. -->
+<!-- 
+            <para>Um festzustellen, ob das Perl-Modul &app-fq; bereits auf Ihrem
+              System installiert ist, geben Sie <command>perldoc Finance::Quote</command> in ein
```

Comment:
Ich glaube, ich würde erstmal
`<prompt>><prompt> <userinpout>perl --version</userinpout>`
(auch unter Windows?) testen. Dann wissen wir, welche Geschmacksrichtung (Erdbeer oder was war das andere?) der Benutzer hat.

Reply:
Ich habe den Block ja noch auskommentiert und den Verweis auf den Guide platziert. Soll ich das wieder zurücknehmen?

Ich bin ja eh der Meinung, dass die Beschreibung zur Installation hier in die Hilfe gehört und nicht in den Guide.

Reply:
Der Meinung bin ich auch, allerdings nicht unbedingt in diesem Kapitel.
Die Hilfe sollte in erster Linie die Dialoge beschreiben und natürlich Verweise dazischen haben.

Vielleicht noch in den Kommentar. Find a proper place.

Reply:
ok.